mros1_reasoner/launch/run.launch

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0"?>
 <launch>
-  <arg name="onto" default="abb_dualarm_mm_complete.owl"/>
+  <arg name="onto" default="kb.owl"/>
+
   <param name="/onto_file" type="string" value="$(arg onto)"/>
```

Comment:
Also out-of-scope, but please do not force parameters in the global scope / use absolute parameter names. This makes it impossible to namespace this `.launch` file.

Reply:
Created issue #86